acapy_plugin_toolbox/holder/v0_1/messages/pres_request_received.py

```diff
@@ -48,7 +48,7 @@ class Fields:
     def __init__(self, record: PresExRecord, **kwargs):
         super().__init__(**kwargs)
         self.raw_repr = record
-        self.presentation_request = record.presentation_request
+        self.presentation_request = record.presentation_request.serialize()
```

Comment:
With the work day nearly over here, I just wanted to mention that this change is what's causing the other tests to fail. For whatever reason, some of the other tests are not setting the `presentation_request` attribute on the `PresExRecord` object, causing the `.serialize()` to fail against the `NoneType` Object Type.